repository/l.json

```diff
@@ -432,6 +432,17 @@
 			]
 		},
 		{
+			"name": "Ledger 3",
+			"details": "https://github.com/praxiq/Ledger3",
+			"labels": ["auto-complete", "language syntax"],
+			"releases": [
+				{
+					"sublime_text": ">=3000",
```

Comment:
This should be `"sublime_text": ">=3084",` since you are using `.sublime-syntax`.